modules/apps/callbacks/types/keys.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+type CallbackType string
+
+const (
+	ModuleName = "ibccallbacks"
+
+	CallbackTypeSendPacket           CallbackType = "send_packet"
+	CallbackTypeAcknowledgement      CallbackType = "acknowledgement"
+	CallbackTypeTimeoutPacket        CallbackType = "timeout"
+	CallbackTypeWriteAcknowledgement CallbackType = "write_acknowledgement"
+
+	// Additional packet data is expected to specify the source callback in the following format
```

Comment:
```suggestion
	// Callback packet data set in the underlying applications packet data is expected to specify 
	// the source callback in the following format
```
sorry formatting is off, but hopefully the idea comes through

Reply:
done